docker-compose.yml

```
@@ -0,0 +1,38 @@
+    container_name: goggles-postgres
+    environment:
+      POSTGRES_DB: goggles
+      POSTGRES_USER: goggles
+      POSTGRES_PASSWORD: goggles
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U goggles -d goggles" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)